File: Course-3/Mobile-dev/Task_0/Base/app/src/main/java/org/hse/android/MainActivity.java
package org.hse.android;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getSupportActionBar().show();

        View button_student = findViewById(R.id.button_student);
        View button_teacher = findViewById(R.id.button_teacher);

        button_student.setOnClickListener(new View.OnClickListener() {
            @Override public void onClick(View v) { Toast.makeText(MainActivity.this, "РАСПИСАНИЕ ДЛЯ СТУДЕНТОВ", Toast.LENGTH_SHORT).show(); }
        });
        button_teacher.setOnClickListener(new View.OnClickListener() {
            @Override public void onClick(View v) { Toast.makeText(MainActivity.this, "РАСПИСАНИЕ ДЛЯ ПРЕПОДОВАТЕЛЯ", Toast.LENGTH_SHORT).show(); }
        });
    }
}

File: Course-3/Mobile-dev/Stepik/7_advanced/7-2/7-2-7/app/src/main/java/com/example/company/myapplication/MainActivity.kt
package com.example.company.myapplication

import android.os.Bundle
import android.os.Environment
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*
import java.io.File
import java.nio.file.FileSystem

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        perform.setOnClickListener {
            var sum = 0
            var p :String = path.text.toString()
            try {

                p = p.replace("\n","").replace(" ","").trim()
                val tost = Toast.makeText(this,"$p", Toast.LENGTH_LONG)
                tost.show()
                var my = File( "$p")

                if (my.isFile) {
                    result.text="Error"
                } else {
                    val array = my.listFiles()

                    for (i in array) {
                        if (i.isFile) sum += 1
                    }
                    result.text = sum.toString()

                }
            } catch (E:NoSuchFileException) {result.text="Error"}
        }
    }
}